pubsub/aws/snssqs/snssqs.go

```diff
@@ -54,17 +55,21 @@ type snsSqsMetadata struct {
 	messageVisibilityTimeout int64
 	// number of times to resend a message after processing of that message fails before removing that message from the queue. Default: 10
 	messageRetryLimit int64
+	// if sqsDeadLettersQueueName is set to a value, then the messageReceiveLimit defines the number of times a message is received
+	// before it is moved to the dead-letters queue. This value must be smaller than messageRetryLimit
+	messageReceiveLimit int64
 	// amount of time to await receipt of a message before making another request. Default: 1
 	messageWaitTimeSeconds int64
 	// maximum number of messages to receive from the queue at a time. Default: 10, Maximum: 10
 	messageMaxNumber int64
 }
 
 const (
-	awsSqsQueueNameKey = "dapr-queue-name"
+	awsSqsQueueNameKey = "dapr-worker-name"
```

Comment:
This change is a breaking change. This component is alpha but I would avoid breaking changes still.

Reply:
sure. you're right